sdk/src/types/block/capabilities.rs

```diff
@@ -58,6 +58,15 @@ impl<Flag> Capabilities<Flag> {
     }
 }
 
+impl<Flag> core::fmt::Debug for Capabilities<Flag> {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("Capabilities")
+            .field("bytes", &self.bytes)
+            .field("flag", &core::any::type_name::<Flag>().rsplit_once("::").unwrap().1)
```

Comment:
For what is this good? Looks pretty useless to me
```
            capabilities: Capabilities {
                bytes: [],
                flag: "TransactionCapabilityFlag",
            },
```

Reply:
It's supposed to show the same information as before, just without the unnecessary fluff.

Reply:
> just without the unnecessary fluff, that reveals Rust details 
But the whole flag field is that, not just exactly the PhantomData?

Reply:
My take on this is that we don't need to print it, because it's clear from the surrounding context, that it's the transaction capabilities, not the address ones, but :man_shrugging:

Reply:
Actually, forget what I said ... I changed it so that it now prints a list of the enabled capabilities (the bytes themselves aren't helpful in the debug, so I removed them from the output).